reference/5-api-publisher/Considerations-docker-configuration-and-execution.md

```diff
@@ -0,0 +1,171 @@
+# Running in Docker Desktop.
+
+Docker Containers have the added benefit of running anywhere (e.g. VMs, on-premises in the cloud), which is a massive advantage for both developers and deployment. Leading cloud providers, including Google Cloud, Amazon Web Services (AWS) and Microsoft Azure have adopted it. For simplicity the steps below describe how to use Docker Compose to deploy the Ed-Fi Api Publisher and related tools on Docker Desktop.
+
+# Setup
+
+## Step 1. Download the Source Code or Clone the Repo
+
+The Ed-Fi ODS Docker deployment source code is in the Ed-Fi repository hosted by GitHub. A link to the repository is provided in the download panel on the right. You can clone the repository or download the source code as a ZIP file.
+
+https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Publisher
+
+## Step 2. Setup Your Environment Variables
+
+Configure your deployments using an environment file. The repository includes a .env.example listing the supported environment variables.
+
+ >                Path:   /src/Compose/env.example
```

Comment:
Is this really intended to be a blockquote? Or is it a code sample that should be in a code fence?

Reply:
fixed